arrays.rb

```diff
@@ -0,0 +1,19 @@
+friends = %w(Shubham Chirag Yuvraj)
+family  = Array[]
```

Comment:
```suggestion
family  = []
```

Reply:
And actually you don't need it since you are re-initializing this variable in next line

Reply:
Done